src/controllers/portfolio/portfolio.ts

```diff
@@ -85,6 +86,14 @@ export class PortfolioController extends EventEmitter {
 
   validTokens: any = { erc20: {}, erc721: {} }
 
+  temporaryTokens: {
+    [networkId: NetworkDescriptor['id']]: {
+      isLoading: boolean
+      errors: ExtendedError[]
```

Comment:
Isn't this error collection heavy for live usage? Are we planning to disable this?  
Also, the ExtendedError is not added as a package in ambire-common. If we plan to use it, we'll need to add it